app/services/ETMPService.scala

```diff
@@ -115,9 +115,15 @@ class ETMPService @Inject()(etmpConnector: ETMPConnector,
     lSPAmountsWithEstimatedStatus + lPPAmountsWithEstimatedStatus
   }
 
-  def getCrystallizedPenaltyAmount(payload: ETMPPayload): Int = {
+  def getNumberOfCrystalizedPenalties(payload: ETMPPayload): Int = {
     val numOfDueLSPs = payload.penaltyPoints.map(_.status).count(status => status == PointStatusEnum.Due)
     val numOfDueLPPs = payload.latePaymentPenalties.getOrElse(Seq.empty).map(_.status).count(status => status == PointStatusEnum.Due)
     numOfDueLSPs + numOfDueLPPs
   }
+
+  def getCrystalisedPenaltyTotal(payload: ETMPPayload):BigDecimal = {
+    val crystallisedLSPAmountDue = payload.penaltyPoints.filter(_.status == PointStatusEnum.Due).map(_.financial.map(_.amountDue).getOrElse(BigDecimal(0))).sum
+    val crystallisedLPPAmountDue = payload.latePaymentPenalties.map(_.filter(_.status == PointStatusEnum.Due).map(_.financial.amountDue).sum).getOrElse(BigDecimal(0))
```

Comment:
This should reference the outstandingAmountDue so we give an accurate amount left in the API:
```suggestion
    val crystallisedLPPAmountDue = payload.latePaymentPenalties.map(_.filter(_.status == PointStatusEnum.Due).map(_.financial.outstandingAmountDue).sum).getOrElse(BigDecimal(0))
```